src/content/about/metrics.md

```diff
@@ -0,0 +1,84 @@
+
+Worker invocation statuses:
+
+| Invocation status      | GraphQL field        | Workers Error code | Definition                                                               |
+| ---------------------- | -------------------- | ------------------ | ------------------------------------------------------------------------ |
+| Success                | success              |                    | Worker script executed successfully                                      |
+| Client Disconnected    | clientDisconnected   |                    | HTTP client (i.e. the browser) disconnected before the request completed |
+| Script Threw Exception | scriptThrewException | 1101               | Worker script threw an unhandled Javascript exception                    |
+| ¹ Exceeded Resources   | exceededResources    | 1102, 1027         | Worker script exceeded runtime limits                                    |
+| ² Internal Error       | internalError        |                    | Workers runtime encountered an error                                     |
+
+¹ The Exceeded Resources status may appear when the Worker exceeds a [runtime limit](/about/limits). The most common cause is excessive CPU time, but is also caused by a script exceeding startup time or free tier limits.
+
+² The Internal Error status may appear when the Workers runtime fails to process a request due to an internal failure in our system. These errors are not caused by any issue with the Worker code nor any resource limit. While requests with Internal Error status are rare, we expect that some may appear during normal operation. These requests are not counted towards usage for billing purposes. If you notice an elevated rate of requests with Internal Error status, please check www.cloudflarestatus.com.
+
+### Requests
+
+This chart shows historical request counts from the Workers runtime broken down into successful requests, failed requests, and subrequests.
+
+- Successful requests: Success and Client Disconnected invocation statuses.
+- Failed requests: Every other invocation status.
+- Subrequests: Requests triggered by calling `fetch` from within a Worker script. A subrequest that throws an uncaught error will not be counted.
+
+### CPU Time
+
+The CPU time chart shows historical CPU time data broken down into relevant quantiles using [reservoir sampling](https://en.wikipedia.org/wiki/Reservoir_sampling). You can learn more about interpreting quantiles [here](https://www.statisticshowto.com/quantile-definition-find-easy-steps/). In some cases, higher quantiles may appear to exceed [CPU time limits](/about/limits/#cpu-execution-time-limit) without generating invocation errors because of a mechanism in the Workers runtime that allows rollover CPU time for requests below the CPU limit.
+
+## Zone Metrics
```

Comment:
It's primarily included to provide context.  If we didn't include it, users might see the zone analytics, visit the docs, and not understand why the analytics don't match.

Reply:
I see what you mean and to be consistent we would add Workers KV also. Neither which I think is a good idea since the UI gives users what they need. To achieve your goal of acknowledging there are 2+ methods of Workers data collected, you don't need a whole section that reiterates what's in the UI. 

You can just add at the top:

>  Workers Metrics are available in Worker script contexts. You can also find Worker analytics per zone in the [dashboard](https://dash.cloudflare.com/?zone=analytics/workers).

Reply:
The info here provides no new info to user, introduces inconsistencies (i.e.  it's not "Metrics" it's under Analytics; it's not in the Workers dashboard it's under a zone's dashboard; it's only shows part of all Workers data), and we plan on not keeping these zone analytics like this. So I feel strongly about not adding it in, not just from a maintainer point of view but also from making users more confused and adding a surplus of content.

Reply:
The main benefit of including this section addresses is discoverability.  Let's say I'm a potential user perusing the documentation for analytics info because I want to compare Cloudflare versus other competitors.

Without the Zone Metrics section, I don't fully understand Cloudflare's product.  I would only know about worker script analytics.  Maybe I would decide to go with the competitor because it didn't meet my needs.

In another case, say I'm a current user trying to find Workers analytics.  I visit the docs and learn they live in the Worker Details page, but I never discover zone analytics because they were difficult to find in Zone > Analytics > Workers, and they weren't listed in the docs.

Isn't it worth adding for these kinds of cases?

Reply:
If the goal is discoverability, then a sentence with the link to the Zone Analytics accomplishes that goal no? Instead of going back and forth let's tag in @donjo

Reply:
Ok, left my thoughts on this here in bullet #2: https://github.com/cloudflare/workers-docs/pull/732#issuecomment-619103306